tensorflow/lite/micro/tools/ci_build/test_x86.sh

```diff
@@ -56,10 +56,13 @@ readable_run make -s -j8 -f tensorflow/lite/micro/tools/make/Makefile integratio
 
 # At last test the hello_world as an example outside of the github repo.
 readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
-cd "../"
-mv tflite-micro/tensorflow/lite/micro/examples/hello_world ./
+pushd "../"
+cp -r tflite-micro/tensorflow/lite/micro/examples/hello_world ./
 sed -i 's/tensorflow\/lite\/micro\/examples\///g' hello_world/Makefile.inc
 sed -i 's/$(TENSORFLOW_ROOT)//g' hello_world/Makefile.inc
 mv hello_world/Makefile.inc hello_world/Makefile_internal.inc
 sed -i 's/tensorflow\/lite\/micro\/examples\///g' hello_world/hello_world_test.cc
 readable_run make -s -j8 -f tflite-micro/tensorflow/lite/micro/tools/make/Makefile test_hello_world_test TENSORFLOW_ROOT=tflite-micro/ EXTERNAL_DIR=hello_world/
+readable_run make -f tflite-micro/tensorflow/lite/micro/tools/make/Makefile clean TENSORFLOW_ROOT=tflite-micro/ EXTERNAL_DIR=hello_world/
+rm -rf hello_world
+popd
```

Comment:
Just a question- Is it preferred to have a 'newline at end of file'?

Reply:
I am not sure but I think that is a good idea. Added the newline.